src/components/media/search_media.graphql

```diff
@@ -5,19 +5,24 @@ query SearchMediaByTitle(
 ) {
   searchMediaByTitle(first: $first, title: $title, mediaType: $media_type) {
     nodes {
-      id
-      type
-      titles {
-        canonical
-      }
-      description(locales: "en")
-      startDate
-      tba
-      posterImage {
-        original {
-          url
-        }
-      }
+      ...media
+    }
+  }
+}
+
+fragment media on Media {
```

Comment:
I think fragment names need to be unique across the app, should make this more descriptive like `mediaSearchFields` or something.

Reply:
Sure. No problem. Will fix tomorrow.

Reply:
Fixed